pyproject.toml

```diff
@@ -6,32 +6,32 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
-[project]
+[tool.poetry]
 name = "gtfs-fetcher"
 version = "0.1.0"
 description = "Download, validate, and manage GTFS transit feeds"
-readme = "README.rst"
-requires-python = ">=3.10"
-license = {file = "LICENSE.txt"}
-authors = [
-  {email = "nils@gis-ops.com"}
-]
-maintainers = [
-  {name = "Nils Nolde", email = "nils@gis-ops.com"}
-]
+authors = ["Nils Nolde <nils@gis-ops.com>"]
```

Comment:
and please add your name/email in the first place:)

Reply:
haha yeah I'll add...